packages/builder/rollup.config.js

```diff
@@ -185,7 +185,11 @@ export default {
     svelte({
       // enable run-time checks when not in production
       dev: !production,
-      include: ["src/**/*.svelte", "node_modules/**/*.svelte"],
+      include: [
+        "src/**/*.svelte",
+        "node_modules/**/*.svelte",
+        "../../../bbui/src/**/*.svelte",
```

Comment:
I had to do this in order for me `yarn link` to bbui to work. It assumes that bbui and budibase are in the same parent directory.